sdk/sdk-ids/src/lib.rs

```diff
@@ -0,0 +1,111 @@
+#![no_std]
+
+pub mod address_lookup_table {
+    solana_pubkey::declare_id!("AddressLookupTab1e1111111111111111111111111");
+}
+
+pub mod bpf_loader {
+    solana_pubkey::declare_id!("BPFLoader2111111111111111111111111111111111");
+}
+
+pub mod bpf_loader_deprecated {
+    solana_pubkey::declare_id!("BPFLoader1111111111111111111111111111111111");
+}
+
+pub mod bpf_loader_upgradeable {
+    solana_pubkey::declare_id!("BPFLoaderUpgradeab1e11111111111111111111111");
+}
+
+pub mod compute_budget {
+    solana_pubkey::declare_id!("ComputeBudget111111111111111111111111111111");
+}
+
+pub mod config {
+    solana_pubkey::declare_id!("Config1111111111111111111111111111111111111");
+}
+
+pub mod ed25519_program {
+    solana_pubkey::declare_id!("Ed25519SigVerify111111111111111111111111111");
+}
+
+pub mod feature {
+    solana_pubkey::declare_id!("Feature111111111111111111111111111111111111");
+}
+
+pub mod loader_v4 {
+    solana_pubkey::declare_id!("LoaderV411111111111111111111111111111111111");
+}
+
+pub mod native_loader {
+    solana_pubkey::declare_id!("NativeLoader1111111111111111111111111111111");
+}
+
+pub mod secp256k1_program {
+    solana_pubkey::declare_id!("KeccakSecp256k11111111111111111111111111111");
+}
+
+pub mod stake {
+    pub mod config {
+        solana_pubkey::declare_deprecated_id!("StakeConfig11111111111111111111111111111111");
+    }
+    pub mod program {
+        solana_pubkey::declare_id!("Stake11111111111111111111111111111111111111");
+    }
```

Comment:
everywhere else, this crate removes the `program` module, so just for consistency with the other programs, can we have `stake::id()`?

```suggestion
    solana_pubkey::declare_id!("Stake11111111111111111111111111111111111111");
```